### DISCUSSION TITLE
Make Alt + Tab only show windows from active monitor

### DISCUSSION THREAD

**Discussion by slrgt** [ramensoftware/windhawk-mods#842] [General]
It would be very convenient if there was a Windhawk mod that allowed us to make Alt + Tab only show windows on monitor that the mouse is on. This option is available on Linux and Mac and it would be great to have it on Windows

**Comment by L3r0yThingz** [comment]
Check this mod: https://windhawk.net/mods/alt-tab-per-monitor.
 I have recently developed a mod for showing the task switcher on the focused display only, check it out! 

![Gif](https://i.imgur.com/Hpg8TKh.gif)

**Reply by slrgt** [reply]
this is amazing thank you!!
